x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/data_provider.ts

```diff
@@ -32,6 +32,7 @@ export interface QueryMatch {
   value: PrimitiveOrArrayOfPrimitives;
   displayValue?: string | number | boolean;
   operator: QueryOperator;
+  params?: Record<string, unknown>;
```

Comment:
this passes arbitrary data down the component tree, so that it can be consumed in hover actions for example